src/Jobs/AfterImportJob.php

```diff
@@ -57,6 +59,13 @@ public function setDependencies(Collection $jobs)
 
     public function handle()
     {
+        // Determine if the batch has been cancelled...
+        if (method_exists($this, 'batchCancelled')) {
```

Comment:
See comment in trait

Reply:
I considered implementing this with a job middleware, but there are versions of Laravel that do not support this, so I discarded the idea.